enterprise-data-seeding/src/seeders/companies_structure_seeder.py

```
@@ -0,0 +1,248 @@
+            )
+            
+        except Exception as e:
+            result.success = False
+            result.errors.append(f"Seeding failed: {str(e)}")
+            logger.exception("Unexpected error during seeding")
+        
+        result.duration_seconds = time.time() - start_time
+        
+        return result
+    
+    async def check_exists_by复合_key(self, model, fields: list[str], *values):
```

Ruff (0.16.2)

[warning] Function name `check_exists_by复合_key` should be lowercase

(N802)